### PR TITLE
SamesiteNone supported by UA - or not

### PR DESCRIPTION
**Pull request by JonathanHuot**
Related to https://github.com/varnishcache/varnish-devicedetect/issues/44

### CODE REVIEW

File: samesite.vcl [hunk 1]
```diff
@@ -0,0 +1,37 @@
+sub samesite {
+        # See list here:
+        # https://www.chromium.org/updates/same-site/incompatible-clients
+        unset req.http.X-UA-SameSiteNone;
+        set req.http.X-UA-SameSiteNone = "supported";
+
+        # Versions of Chrome from Chrome 51 to Chrome 66 (inclusive on both ends). These Chrome versions will reject a cookie with `SameSite=None`
+        if (req.http.user-agent ~ "Chrom(e|ium)" &&
```
Comment (gquintard) [inline samesite.vcl:8]: no need to test the header existence

Comment (JonathanHuot) [reply samesite.vcl:8]: Hi Guillaume, are you talking about the VCL or VTC part ?

Comment (gquintard) [reply samesite.vcl:8]: hum, I must have misread that code, I was sure it was
```
if (req.http.user-agent && req.http.user-agent ~ "Chrom(e|ium)" &&
```

nevermind me...

**Comment by JonathanHuot** [comment]
Any updates on this ?

**Comment by gquintard** [comment]
Hi @JonathanHuot, yes, it looks ok, we should merge that soon (the maintainer is currently on vacation though)

**Pull request merged**